### PR TITLE
ZEPPELIN-412 Documentation based on Zeppelin version (website)

### PR DESCRIPTION
**Pull request by Leemoonsoo**
Together with #430, it addresses https://issues.apache.org/jira/browse/ZEPPELIN-412

* gh-pages branch will serve document version independent part of the website.
* per version documentation will be served by 'docs' directory in main source tree.

Following is screenshot of website from this PR.

image

### CODE REVIEW

**Comment by jongyoul** [comment]
The screenshot shows what I expect exactly. LGTM

File: 404.html [hunk 1]
```diff
@@ -0,0 +1 @@
+Sorry this page does not exist =(
```
Comment (felixcheung) [inline 404.html]: gh page doesn't have this by default?

**Comment by felixcheung** [comment]
is there a `'latest`?
https://zeppelin.incubator.apache.org/docs/latest/index.html

File: 404.html (hunk 1 above)
Comment (Leemoonsoo) [inline 404.html]: Even though we use branch name gh-pages, actual deployment goes into https://svn.apache.org/repos/asf/incubator/zeppelin. So i think it is irrelevant to github-pages.

**Comment by Leemoonsoo** [comment]
To provide 'latest', i think symbolic link need to be made. In deployment side, I'm not sure if i can push symlink to https://svn.apache.org/repos/asf/incubator/zeppelin.

When i deploy this change, let me try to make symlink 'latest'.

File: 404.html (hunk 1 above)
Comment (felixcheung) [inline 404.html]: ah I see.

**Pull request closed without merging**